types.go

```diff
@@ -0,0 +1,12 @@
+package celeriac
+
+import "time"
+
+// Message defines the properties of the payload which transported
+// through the queue
+type Message struct {
+	Expiration string
```

Comment:
Should `Expiration` be a `time.Time`?

Reply:
Have updated to use `*time.Time`  to facilitate scenarios where there may not be an expiration